ddsketch/mapping.py

```diff
@@ -16,15 +16,17 @@
 when computing the index. Other mappings can approximate the logarithmic
 mapping, while being less computationally costly.
 """
-from abc import ABC
```

Comment:
should we import `__division__` anyways just to be sure it is compatible with py3?

Reply:
yeah good idea, i thought about it and then forgot